bot/cogs/moderation/silence.py

```diff
@@ -141,6 +162,7 @@ async def _unsilence(self, channel: TextChannel) -> bool:
             await channel.set_permissions(self._verified_role, **dict(current_overwrite, send_messages=None))
             log.info(f"Unsilenced channel #{channel} ({channel.id}).")
             self.notifier.remove_channel(channel)
+            self.cancel_task(channel.id)
             self.muted_channels.discard(channel)
```

Comment:
I feel like it would look better if the cancel task came before or after the `.remove_channel` and `.discard`, as those do the same job and make sense to be together